### PR TITLE
docs(std): version all imports in README

### PR DESCRIPTION
**Pull request by tokiedokie**
fixes

> All imports in readme examples should be versioned, and bumped on release

in #6844

### CODE REVIEW

**Comment by caspervonb** [comment]
Tag as `docs(std)`.

**Review by crypt096 — APPROVED** [review]

File: std/flags/mod.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export interface ArgParsingOptions {
    * the result `['--']` with everything after the `--`. Here's an example:
    *
    *      // $ deno run example.ts -- a arg1
-   *      import { parse } from "https://deno.land/std/flags/mod.ts";
+   *      import { parse } from "https://deno.land/std@0.70.0/flags/mod.ts";
```
Comment (bartlomieju) [inline std/flags/mod.ts:18]: Imports from docstrings should be removed - they will get outdated with each new release

File: std/archive/README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 ## Tar
 
 ```ts
-import { Tar } from "https://deno.land/std/archive/tar.ts";
+import { Tar } from "https://deno.land/std@0.70.0/archive/tar.ts";
```
Comment (bartlomieju) [inline std/archive/README.md:6]: @tokiedokie this is nice, but the problem is we'd have to update it with each release and we're already at `0.71.0`. I chatted with @lucacasonato and we could handle this more gracefully by using `$STD_VERSION` tag in `std`'s READMEs (similarly to files in `docs/` directory).

**Review by bartlomieju — CHANGES_REQUESTED** [review]

File: std/flags/mod.ts (hunk 1 above)
Comment (tokiedokie) [reply std/flags/mod.ts]: done

File: std/archive/README.md (hunk 2 above)
Comment (tokiedokie) [reply std/archive/README.md]: done

**Comment by bartlomieju** [comment]
Blocked by https://github.com/denoland/deno_website2/issues/1542

**Comment by tokiedokie** [comment]
https://github.com/denoland/deno_website2/issues/1542 was closed via https://github.com/denoland/deno_website2/pull/1551

ready to merge

File: std/datetime/README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ are supported:
 Takes an input `string` and a `formatString` to parse to a `date`.
 
 ```ts
-import { parse } from 'https://deno.land/std/datetime/mod.ts'
+import { parse } from 'https://deno.land/std@0.69.0/datetime/mod.ts'
```
Comment (bartlomieju) [inline std/datetime/README.md:40]: hardcoded version

File: std/datetime/README.md [hunk 4]
```diff
@@ -54,7 +54,7 @@ parse("01-20-2019 16:34:23.123", "MM-dd-yyyy HH:mm:ss.SSS") // output : new Date
 Takes an input `date` and a `formatString` to format to a `string`.
 
 ```ts
-import { format } from "https://deno.land/std/datetime/mod.ts";
+import { format } from "https://deno.land/std@0.69.0/datetime/mod.ts";
```
Comment (bartlomieju) [inline std/datetime/README.md:57]: ditto

**Review by bartlomieju — CHANGES_REQUESTED** [review]

File: std/datetime/README.md (hunk 3 above)
Comment (tokiedokie) [reply std/datetime/README.md:40]: sorry, I missed it.

**Review by bartlomieju — APPROVED** [review]
LGTM, thanks @tokiedokie, great contribution

**Comment by KotlinIsland** [comment]
But now the readme code samples on Github display `$STD_VERSION` and don't work.

**Comment by tokiedokie** [comment]
@KotlinIsland Which README are you taking about?

**Comment by KotlinIsland** [comment]
[Project readme](https://github.com/denoland/deno/blob/master/README.md)

**Comment by tokiedokie** [comment]
@KotlinIsland This PR versions std's README.
The README you mentioned is not maintenanced in this PR,  so you should open a new issue or PR.

**Comment by KotlinIsland** [comment]
That readme is the first change listed in this PR [here](https://github.com/denoland/deno/pull/7442/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R68)  

[Github](https://github.com/denoland/deno#getting-started)
image
[deno.land](https://deno.land/#getting-started)
image

This is extremely confusing to someone who don't understand what this means.

**Pull request merged**